sdcm/utils/common.py

```diff
@@ -1951,3 +1951,26 @@ def ec2_ami_get_root_device_name(image_id, region):
             return image.root_device_name
     except (TypeError, ClientError):
         raise AssertionError(f"Image '{image_id}' details not found in '{region}'")
+
+
+def get_snapshot_content(snapshot_name, listsnapshots_output) -> list():
+    """
+    listsnapshots output:
+
+        Snapshot Details:
+        Snapshot name Keyspace name Column family name True size Size on disk
+        1599414845162 system_schema keyspaces          0 bytes   71.71 KB
+        1599414845162 system_schema scylla_tables      0 bytes   73.21 KB
+        1599414845162 system_schema tables             0 bytes   81.48 KB
+        1599414845162 system_schema columns            0 bytes   80.12 KB
+
+        Total TrueDiskSpaceUsed: 0 bytes
+    """
+    snapshot_content = []
+    for line in listsnapshots_output.stdout.splitlines():
```

Comment:
expecting unneeded objects and calling them "output" is a bad SW dev practice
`listsnapshots_output.stdout` -> `listsnapshots_output`

Reply:
> expecting unneeded objects and calling them "output" is a bad SW dev practice
> `listsnapshots_output.stdout` -> `listsnapshots_output`

I don't understand. Maybe do you mean:
```listsnapshots_output.stdout -> listsnapshots.stdout```

Reply:
Fixed